### PR TITLE
Next generation UI framework

### PR DESCRIPTION
**Pull request by berggren**
This PR is the beginning of a new UI. The goal is to create an easier to develop UI, as well as improving consistency in UI workflows.

Technically we are moving to Vuetify as the UI framework. It is based on the material UI specs, and is a bit more opiniated than our current solution with BulmaCSS + Buefy.

The port to Vuetify will take many iterations and PRs, but this is the humble beginning of that.

Example screenshot
Screenshot 2022-01-25 at 17 11 48

### CODE REVIEW

File: docker/release/build/Dockerfile-latest [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Use the official Docker Hub Ubuntu 20.04 base image
 FROM ubuntu:20.04
 
-MAINTAINER Timesketch <timesketch-dev@googlegroups.com>
```
Comment (jaegeral) [inline docker/release/build/Dockerfile-latest:4]: Not sure why this is removed here

Comment (jaegeral) [inline docker/release/build/Dockerfile-latest:4]: Not sure why this is removed here

Comment (jaegeral) [inline docker/release/build/Dockerfile-latest:4]: Not sure why this is removed here

File: docker/release/docker-compose.yml [hunk 2]
```diff
@@ -13,7 +12,6 @@ services:
       - ./logs:/var/log/timesketch/
 
   timesketch-worker:
```
Comment (jaegeral) [inline docker/release/docker-compose.yml:14]: Not sure why the container_names should be removed in this PR. If they need to be removed I opt to make it in a dedicated PR so it is traceable

Comment (jaegeral) [reply docker/release/docker-compose.yml:14]: same for the container names below

File: timesketch/frontend-ng/src/App.vue [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-app id="app">
+    <v-app-bar app clipped-right flat :color="$vuetify.theme.dark ? '' : 'white'">
+      <v-img src="/dist/timesketch-color.png" max-height="30" max-width="30" contain></v-img>
+      <v-toolbar-title class="ml-3"> timesketch </v-toolbar-title>
```
Comment (jaegeral) [inline timesketch/frontend-ng/src/App.vue:20]: s/timesketch/Timesketch ?

File: timesketch/frontend-ng/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Timesketch web frontend
```
Comment (jaegeral) [inline timesketch/frontend-ng/README.md:1]: can we move that file to the `/docs/` folder instead?

File: timesketch/frontend-ng/src/App.vue (hunk 3 above)
Comment (berggren) [reply timesketch/frontend-ng/src/App.vue:20]: No, this is part of the logo, and it is lowercase (same as today) :)

File: timesketch/frontend-ng/src/App.vue [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn icon v-on:click="toggleTheme" v-bind="attrs" v-on="on">
+            <v-icon>mdi-brightness-6</v-icon>
+          </v-btn>
+        </template>
+        <span>Switch between light and dark theme</span>
+      </v-tooltip>
+
+      <v-avatar class="ml-3" color="orange" size="32">
+        <span class="white--text">jb</span>
```
[the remark this answers is not in the document]
Comment (berggren) [reply timesketch/frontend-ng/src/App.vue:36]: Just a placeholder - we don't have support for avatars yet, so this will be generated from the username for now.

File: timesketch/frontend-ng/README.md (hunk 4 above)
Comment (berggren) [reply timesketch/frontend-ng/README.md:1]: Not at this time - this is meant as a readme for anyone (dev) who are browsing the repo. I'll write more appropriate docs at a later stage

File: docker/release/docker-compose.yml (hunk 2 above)
Comment (berggren) [reply docker/release/docker-compose.yml]: good catch. I have reverted back.

File: docker/release/build/Dockerfile-latest (hunk 1 above)
Comment (berggren) [reply docker/release/build/Dockerfile-latest:4]: The `MAINTAINER` directive has been deprecated in docker-compose, thats why we have removed it.

**Comment by berggren** [comment]
@jaegeral PTAL

File: timesketch/frontend-ng/README.md (hunk 4 above)
Comment (jaegeral) [reply timesketch/frontend-ng/README.md:1]: ACK

**Comment by jaegeral** [comment]
LGTM
